src/api/telemetry/BatchingWebSocket.js

```diff
@@ -74,8 +63,8 @@ class BatchingWebSocket extends EventTarget {
     this.#worker = new Worker(workerUrl);
     this.#openmct = openmct;
     this.#showingRateLimitNotification = false;
-    this.#maxBatchSize = Number.POSITIVE_INFINITY;
-    this.#maxBatchWait = ONE_SECOND;
+    this.#maxBufferSize = Number.POSITIVE_INFINITY;
+    this.#throttleRate = ONE_SECOND;
```

Comment:
Should this be configurable per deployment?

Reply:
[It is](https://github.com/nasa/openmct/pull/7837/files/4126b21bf41bd385cf99c300d7431a5b3c93854a#diff-ad7bb0f2f80ab9afc9c3eadbe47859ed40094b6a487141fd2a72bffeb216e62cR135), this is just assigning a default value.